dist/images/ovnkube.sh

```diff
@@ -48,6 +48,8 @@
 # OVN_LOG_SB - log level (ovn-ctl default: -vconsole:off -vfile:info) - v3
 # OVN_LOG_CONTROLLER - log level (ovn-ctl default: -vconsole:off -vfile:info) - v3
 # OVN_LOG_NBCTLD - log file (ovn-nbctl daemon mode default: /var/log/openvswitch/ovn-nbctl.log)
+# OVN_EMPTY_LB_EVENTS - boolean indicating if ovnkube should raise an event when a packet is sent
```

Comment:
Should we just make this `OVN_ENABLE_UNIDLING` since that's what it's really doing? @girishmg

Reply:
I've left this as-is for now. I'm not certain how well the term "unidling" translates outside OpenShift. If I'm wrong then let me know and I'll make the switch.